e2e/inference/README.md

```diff
@@ -29,10 +29,26 @@ Follow this [link](https://github.com/openvinotoolkit/operator/blob/v1.1.0/docs/
 ## Work with interactive mode
 To enable the interactive mode, the OpenVINO notebook CR needs to be created and integrated with RHODS.  
 1.	Click on the ```create Notebook``` option in this [link](https://github.com/red-hat-data-services/odh-deployer) from the web console and follow these [steps](https://github.com/openvinotoolkit/operator/blob/main/docs/notebook_in_rhods.md) to create the notebook CR.
-2.	Enable Intel Data Center GPU 
+2.	Enable Intel Data Center GPU - **Technical Preview feature**
+
+Create AcceleratoProfile in the ```redhat-ods-applications``` namespace 
+
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/e2e/inference/accelerator_profile.yaml```
+
+3. Navigate to ```openvino-notebooks``` ImageStream and add the above created ```AcceleratorProfile``` key to the annotation field, as shown in the image below:
+
+![Alt text](/docs/images/openvino-accelerator-field.png)
+
+4. Access ```rhods-dashboard``` route in the ```redhat-ods-applications``` namespace as in the image below
+   
```

Comment:
Nativegate to ... in the image blow. Click the location link, the RHODS Dashboard can be launched.

Reply:
We can actually remove this part, we already mention it here https://github.com/intel/intel-technology-enabling-for-openshift/tree/main/e2e/inference#verification

Reply:
I still prefer to keep it here, since it is not a verification, it is a must-have step to use Intel GPU.

Reply:
updated